src/components/11-infographics/chart.njk

```diff
@@ -0,0 +1,35 @@
+<div class="usa-chart" data-chart-type="bar"> <!-- char-type: bar, line -->
+  <table class="usa-chart-data" data-min="0" data-max="100" data-increment="20" data-grid="true"> <!-- all values optional -->
+
```

Comment:
Can you add a `<caption>` as the title giving a brief description of the table? See: https://webaim.org/techniques/tables/data#caption